skyrim-platform/src/third_party/cef/ui/MyChromiumApp.cpp

```diff
@@ -98,11 +98,9 @@ void MyChromiumApp::Initialize() noexcept
   CefWindowInfo info;
   info.SetAsWindowless(m_pRenderProvider->GetWindow());
 
-  if (!CefBrowserHost::CreateBrowser(
-        info, m_pGameClient.get(),
-        L" "
-        L"skymp-gamemode-outdated/front/chat.html",
-        browserSettings, nullptr, nullptr)) {
+  if (!CefBrowserHost::CreateBrowser(info, m_pGameClient.get(),
+                                     L"file:///Data/Platform/UI/index.html",
```

Comment:
Appears that for SkyMP it'll first blink with 404 page before loading chat from http server

Reply:
not used in skymp for a while. not a valid URL at all: " skymp-gamemode-outdated/front/chat.html". there is a whitespace character on the prev line